src/plugins/GUITestBase/src/GUITestBasePlugin.cpp

```diff
@@ -1427,6 +1427,7 @@ void GUITestBasePlugin::registerTests(UGUITestBase *guiTestBase) {
     REGISTER_TEST(GUITest_regression_scenarios::test_5447_1);
     REGISTER_TEST(GUITest_regression_scenarios::test_5447_2);
     REGISTER_TEST(GUITest_regression_scenarios::test_5447_3);
+    REGISTER_TEST_ONLY_WINDOWS(GUITest_regression_scenarios::test_5450);
```

Comment:
Why do you register this test on Windows only? Could you please add it to all nightly builds?

Everything else looks good.